docs/modules/user-guide/pages/adding-volume-component.adoc

```diff
@@ -0,0 +1,6 @@
+:description: Adding volume component to a devfile
```

Comment:
Do these attributes normally include or omit articles? For example, should this be:
```suggestion
:description: Adding a volume component to a devfile
```

Reply:
Or something more like 
```suggestion
:description: Adding volume components to devfiles
```